suite-common/wallet-config/src/networksConfig.ts

```diff
@@ -1,5 +1,101 @@
 import { DeviceModelInternal } from '@trezor/connect';
-import type { Keys, Without } from '@trezor/type-utils';
+import type { Without } from '@trezor/type-utils';
+
```

Comment:
I intend to migrate the types to separate file later. Now it would create a circular dependency..